### PR TITLE
Does this make it work for you?

### PR DESCRIPTION
**Pull request by anmonteiro**

### CODE REVIEW

File: pkg1/dune-workspace [hunk 1]
```diff
@@ -1,2 +1 @@
 (lang dune 3.2)
-(context (opam (switch default)))
```
Comment (anmonteiro) [inline pkg1/dune-workspace:2]: I don't have opam installed and it was complaining.

File: pkg2/lib/bar.ml [hunk 2]
```diff
@@ -1,3 +1,3 @@
-open Foo
+open Pkg1.Foo
```
Comment (anmonteiro) [inline pkg2/lib/bar.ml:1]: Dune wraps libraries in the `public_name`. you can use `(wrapped false)` if you want the toplevel namespace to be `Foo`. Or alternatively name the library `foo`

**Comment by jgertm** [comment]
Thank you Tony, it does indeed work for me this way! 

Seems like all that was fundamentally required (as also pointed out by https://github.com/ocaml/dune/issues/6358#issuecomment-1295965673) was to remove the `context` stanza.

Your renaming of the import makes sense to me.

Do you think this is sufficiently documented? I was not aware of this even after poring over the Dune manual for days.

**Comment by anmonteiro** [comment]
> Do you think this is sufficiently documented?

I think it's not sufficiently documented. This is also not about dune, but rather findlib / ocamlfind (which dune wraps / reimplements in some areas). The combination of "who's doing what" can be hard to grasp.

Personally I've only learned about this stuff by digging into it. I've learned a lot by packaging OCaml for Nix, where they use findlib in a rather non-traditional way ([setupHook](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/ocaml/findlib/default.nix#L30)).

**Pull request closed without merging**